backend/spec/requests/api/v1/accounts/projects_spec.rb

```diff
@@ -62,6 +62,76 @@
   }
 
   path "/api/v1/account/projects" do
+    get "Returns list of projects of User" do
+      tags "Projects"
+      produces "application/json"
+      security [csrf: [], cookie_auth: []]
+
+      parameter name: "fields[project]", in: :query, type: :string, description: "Get only required fields. Use comma to separate multiple fields", required: false
+      parameter name: :includes, in: :query, type: :string, description: "Include relationships. Use comma to separate multiple fields", required: false
+      parameter name: "filter[full_text]", in: :query, type: :string, required: false, description: "Filter records by provided text."
+      parameter name: :sorting, in: :query, type: :string, required: false, description: "Sort records.",
+        enum: [
+          "name asc", "name desc",
+          # "category asc", "category desc", # TODO: this is stored as a slug in the db, i.e. in English
+          # "municipality asc", "municipality desc", # TODO: this shows as 'Leticia, Colombia' in the design
+          # "ticket_size asc", "ticket_size desc", # TODO: this is stored as a slug in the db, i.e. in English
+          # "instrument_type asc", "instrument_type desc", # TODO: this is an array of slugs
+          # "status asc", "status desc", # TODO: this is stored as a slug in the db, i.e. in English
+          "created_at asc", "created_at desc"
```

Comment:
Hmm, I think that to cover all these sorting options with `API::Sorter` is gonna be tricky. Maybe we could rewrite this service to use ransack, which Admin is already using and just add some transformation table which translates FE option to ransack params. Difficult part is that `API::Sorter` is used for sorting everything (PDs/Projects/etc.) so preparation of ransack params would need to take this into account and tweak attributes appropriately --> for example `name` is localized for Projects, but not localized for Accounts, etc.

Reply:
Yeah, I think using ransack in or instead of `API::Sorter` could make things easier.